cmd/serve.go

```diff
@@ -119,7 +120,9 @@ func serve() {
 		}
 	}
 
-	router := web.NewRouter(idpConfig, schemasConfig, rulesConfig, extCfg, web.NewO11yConfig(tracer, monitor, logger))
+	wpool := pool.NewWorkerPool(specs.WorkersTotal, tracer, monitor, logger)
+
+	router := web.NewRouter(idpConfig, schemasConfig, rulesConfig, extCfg, web.NewO11yConfig(tracer, monitor, logger), wpool)
```

Comment:
```suggestion
	router := web.NewRouter(idpConfig, schemasConfig, rulesConfig, extCfg,wpool,  web.NewO11yConfig(tracer, monitor, logger))
```

leave o11y config as the last one in any signature (implicit convention)